README.md

```diff
@@ -46,6 +46,9 @@ If you use any [functions](https://github.com/krzkaczor/ts-essentials/blob/maste
     - DeepNullable
     - DeepUndefinable
     - [Difference between `DeepRequired` and `DeepNonNullable`](#difference-between-deeprequired-and-deepnonnullable)
+  - [String literal types](#string-literal-types)
```

Comment:
I think this is missing readme entry

Reply:
Ah, right

Let me add the description before merging

Reply:
Added a separate README for literal types ✅